pootle/runner.py

```diff
@@ -124,13 +128,24 @@ def run_app(project, default_settings_path, settings_template,
                 print("File already exists, not overwriting.")
                 exit(2)
 
+        if args.db in ["mysql", "postgresql"]:
+            settings_template = "%s.%s" % (settings_template, args.db)
+        elif args.db != "sqlite":
+            raise management.CommandError("Unrecognised database '%s': should "
+                                          "be one of sqlite, mysql or "
+                                          "postgresql" % args.db)
+
         try:
             init_settings(config_path, settings_template)
         except (IOError, OSError) as e:
             raise e.__class__('Unable to write default settings file to %r'
                 % config_path)
 
-        print("Configuration file created at %r" % config_path)
+        if args.db in ["mysql", "postgresql"]:
+            print("Configuration file created at %r: you must now update "
+                  "the settings for %s database" % (config_path, args.db))
```

Comment:
Perhaps it is better to also ask for those details to be passed as arguments to `pootle init`. That way this can be completely automated.